<launch>

  <arg name="robot_namespace" default="robot"/>

  <arg name="path" default="/home/nakul/Desktop/vectr/ws/src/" />
  <arg name="file_1" default="bag_3.bag" />

  <!-- DLIO Odometry Node -->
  <node ns="$(arg robot_namespace)" name="map_align" pkg="direct_lidar_inertial_odometry" type="map_align_node" output="screen" clear_params="true">

    <!-- Load parameters -->
    <rosparam file="$(find direct_lidar_inertial_odometry)/cfg/map_align_params.yaml" command="load"/>

  </node>

  <node pkg="rosbag" type="play" name="player" output="screen" args="--clock $(arg path)$(arg file_1)"/>

</launch>